src/components/content/order/migrate/Migrate.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+    CloudServiceProvider,
+    ServicesAvailableService,
+    ServiceVo,
+    UserAvailableServiceVo,
+} from '../../../../xpanse-api/generated';
+import { SelectDestination } from './SelectDestination';
+import { ShowDeploy } from './ShowDeploy';
+import { Steps } from 'antd';
+import { MigrateService } from './MigrateService';
+import { MigrationSteps } from '../formElements/CommonTypes';
+import { ExportServiceData } from './ExportServiceData';
+import { ImportServiceData } from './ImportServiceData';
+
+export const Migrate = ({
+    currentSelectedService,
+    getMigrateModalOpenStatus,
+}: {
+    currentSelectedService: ServiceVo | undefined;
+    getMigrateModalOpenStatus: (isMigrateModalOpen: boolean) => void;
+}): JSX.Element => {
+    const [currentMigrationStep, setCurrentMigrationStep] = useState<MigrationSteps>(MigrationSteps.ExportServiceData);
+    const [currentMigrationStepStatus, setCurrentMigrationStepStatus] = useState<
+        'wait' | 'process' | 'finish' | 'error' | undefined
```

Comment:
values must be processing, finished, failed.

not sure if `wait` is really used. if yes, then it must be called 'waiting'